apps/iframe/src/services/connectService.ts

```diff
@@ -10,14 +10,11 @@ import ThreeIdProvider from '3id-did-provider'
 import type { DIDMethodName, DIDProvider, DIDProviderMethods, DIDRequest, DIDResponse } from 'dids'
```

Comment:
@zachferland do we need this module here please? I thought I asked already in a previous version, I guess I still don't get why this logic is part for the UI prompts?